src/interface/web/app/components/suggestions/suggestionCard.tsx

```diff
@@ -81,46 +81,41 @@ function getIconFromIconName(iconName: string, color: string = 'gray', width: st
 }
 
 function convertSuggestionColorToIconClass(color: string) {
-    if (color.includes('sky')) return getIconFromIconName('Robot', 'blue', 'w-8', 'h-8');
+    if (color.includes('blue')) return getIconFromIconName('Robot', 'blue', 'w-8', 'h-8');
     if (color.includes('yellow')) return getIconFromIconName('Globe', 'yellow', 'w-8', 'h-8');
     if (color.includes('green')) return getIconFromIconName('Palette', 'green', 'w-8', 'h-8');
     else return getIconFromIconName('Lightbulb', 'orange', 'w-8', 'h-8');
 }
 
 
 interface SuggestionCardProps {
-	title: string;
-	body: string;
-	link: string;
-	image: string;
-	color: string;
+    title: string;
+    body: string;
+    link: string;
+    image: string;
+    color: string;
 }
 
 export default function SuggestionCard(data: SuggestionCardProps) {
-    if (data.link !== "") {
-        return (
-            <a href={data.link} className="no-underline">
-            <Card className={`${styles.card} ${data.color}`}>
-                <CardHeader className="m-0 p-2 pb-1 relative">
-                    {convertSuggestionColorToIconClass(data.image)}
-                    <CardTitle className={`${styles.title} pt-2 dark:text-black`}>{data.title}</CardTitle>
-                </CardHeader>
-                <CardContent className="m-0 p-2 pr-4 pt-1 w-[200px] h-[100px]">
-                    <CardDescription className={styles.text}>{data.body}</CardDescription>
-                </CardContent>
-            </Card>
-            </a>
-        );
-    }
-    return (
-        <Card className={`${styles.card} ${data.color}`}>
-            <CardHeader className="m-0 p-2 pb-1 relative">
-                {convertSuggestionColorToIconClass(data.image)}
-                <CardTitle className={`${styles.title} pt-2 dark:text-black`}>{data.title}</CardTitle>
-            </CardHeader>
-            <CardContent className="m-0 p-2 pr-4 pt-1 w-[200px] h-[100px]">
-                <CardDescription className={styles.text}>{data.body}</CardDescription>
-            </CardContent>
+    const cardClassName = `${styles.card} ${data.color} md:w-full md:h-fit sm:w-full sm:h-fit lg:w-[200px] lg:h-[200px]`;
+    const titleClassName = `${styles.title} pt-2 dark:text-white dark:font-bold`;
+    const descriptionClassName = `${styles.text} dark:text-white`;
+
+    const cardContent = (
```

Comment:
Would generally be good to right click in your VSCode and run format document to avoid the spacing/tabbing issues. Ideally this should be part of CI, but we haven't set it up yet.

<img width="271" alt="Screenshot 2024-07-23 at 13 08 20" src="https://github.com/user-attachments/assets/45670c79-07aa-4510-89c0-5d7f91a0bbc1">